engines/twine/detection.cpp

```diff
@@ -550,6 +644,29 @@ class TwinEMetaEngineDetection : public AdvancedMetaEngineDetection {
 	const char *getOriginalCopyright() const override {
 		return "Little Big Adventure (C) Adeline Software International";
 	}
+
+	const ExtraGuiOptions getExtraGuiOptions(const Common::String &target) const override;
 };
 
+const ExtraGuiOptions TwinEMetaEngineDetection::getExtraGuiOptions(const Common::String &target) const {
+	ExtraGuiOptions options;
+	options.push_back(OptWallCollision);
+	options.push_back(OptCrossFade);
+	options.push_back(OptDisableSaveMenu);
+	options.push_back(OptMouse);
+	options.push_back(OptHighRes);
+	options.push_back(OptSound);
+	options.push_back(OptUseCD);
+	// TODO: only 7 are shown right onw - see GUI::ExtraGuiOptionsWidget
```

Comment:
I think this comment doesn't apply anymore. But I can remove it later.

Reply:
Indeed,  this no longer applies